desktop/apps/artwork/components/auction_artworks/helpers.coffee

```diff
@@ -13,8 +13,16 @@ zone = (time) ->
   moment(time).tz('America/New_York')
 
 module.exports =
-  masonry: (artworks) ->
-    masonry take shuffle(artworks), ARTWORK_DISPLAY_NUM
+  masonry: (artworks, followed_artist_ids) ->
+    followIds = pluck followed_artist_ids.hits, 'id'
+
+    # Find followed artists and prepare to prepend to results array
+    [followed, rest] = partition artworks, (artwork) =>
+      followIds.some (id) =>
+        artwork.id == id
+
```

Comment:
minor and maybe breaks convention, but i feel like the anonymous function on L21-22 is so trivial that putting it on a single line would make the overall `partition` call more readable. If you feel differently though I'd trust you.